catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/Gml3ToWktImpl.java

```diff
@@ -38,15 +32,34 @@
 import org.slf4j.LoggerFactory;
 import org.xml.sax.SAXException;
 
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Collections;
+
 public class Gml3ToWktImpl implements Gml3ToWkt {
   private static final Logger LOGGER = LoggerFactory.getLogger(Gml3ToWkt.class);
 
   private static final String EPSG_4326 = "EPSG:4326";
 
-  private final Parser parser;
+  protected Parser parser;
 
   private static final ThreadLocal<WKTWriter> WKT_WRITER = ThreadLocal.withInitial(WKTWriter::new);
 
+  public Gml3ToWktImpl() {
```

Comment:
Geotools uses Java SPI. When instantiated from blueprint SPI would get a different classloader than expected and failed to find its services. I had to break the implementation out into two separate classes since we couldn't use blueprint injection.